app/views/likes/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for([@image, @like]) do |f| %>
+
+<%= hidden_field_tag :user_id, current_user.id %>
```

Comment:
No need to put this in the form since you have implicit access to it in the controller.